src/sdk/sdkSession.ts

```diff
@@ -256,7 +256,7 @@ class SDKSession {
         // in localStorage because getBtcWalletData is also called in the constructor)
         this.deviceID = deviceID;
         this.getBtcWalletData();
-        return cb(null, isPaired);
+        return cb ? cb(null, isPaired) : isPaired;
```

Comment:
Are you sure you shouldn't return `null` instead of `isPaired`?

Reply:
The first arg is the value of `err`, second is the expected value (in this case `isPaired`), so if we aren't using the callback syntax, we just want to return the expected value. 

I'm going to revise this in the future to remove the callback stuff entirely since it _shouldn't_ be used anywhere, but I wanted to get this fix out asap.